platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -702,17 +704,15 @@ clients:
   #     allowAnyDestination: false
   #     destinationAccounts:
   #       - GDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
```

Comment:
nit: use a string that matches the length of a public key.  `GDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD`